lib/game.rb

```diff
@@ -0,0 +1,10 @@
+class Rps
+  attr_reader :guess
+
+  def initialize
+
+
+  end
+
```

Comment:
This looks like a great start. 

What data do you think this class needs to know about?
What methods do you think it should have?